server/utils/nunjucksSetup.ts

```diff
@@ -31,6 +31,9 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
   app.locals.asset_path = '/assets/'
   app.locals.applicationName = 'Digital Prison Services'
 
+  app.locals.applicationInsightsConnectionString = config.applicationInsights.connectionString
```

Comment:
Make the app insights connection string, and the application name, available to `app.locals` so that nunjucks can reference them in the template